### PR TITLE
Cryptomari0 binance 5

### PR DESCRIPTION
**Pull request by cryptomari0**

### CODE REVIEW

**Review by coinmandeer-zz — CHANGES_REQUESTED** [review]

File: db/projects/neblio/project.yaml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    photo: https://nebl.io/wp-content/uploads/2017/09/riley-e1505066565129-250x250.jpg
+    webids:
+      linkedin: stocktonriley
+  - name: Samer Afach
+    role: developer
+    photo: https://nebl.io/wp-content/uploads/2017/12/sam-250x250.jpg
+    webids:
+      linkedin: samerafach
+whitepapers:
+  - title: 'neblio: Next Generation Enterprise Blockchain Solutions'
```
[the remark this answers is not in the document]
Comment (cryptomari0) [reply db/projects/neblio/project.yaml]: - title: 'Neblio: Next Generation Enterprise Blockchain Solutions'

[the remark this answers is not in the document]
Comment (cryptomari0) [reply db/projects/neblio/project.yaml]: description: 'A secure, distributed, platform built for enterprise applications and services. Running on the Neblio blockchain network.'

[the remark this answers is not in the document]
Comment (cryptomari0) [reply db/projects/neblio/project.yaml]: tags:
  - business
  - enterprise
  - blockchain

File: db/projects/nuls/project.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+name: Nuls
+web:
+  - 'https://nuls.io/'
+assets:
```
[the remark this answers is not in the document]
Comment (cryptomari0) [reply db/projects/nuls/project.yaml]: token_standard: ERC-20

File: db/projects/ost/project.yaml [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  github: OpenSTFoundation
+  twitter: OSTdotcom
+  reddit: OSTdotcom
+  facebook: OST
+  youtube/channel: 'UCrDo4vSElgUa_HzdvYqBZTA'
+  custom/forum: https://discuss.openst.org/
+  gitter: OpenSTFoundation
+contacts:
+  email: 'review@ost.com'
+  location:hq: 'Berlin, Germany'
```
[the remark this answers is not in the document]
Comment (cryptomari0) [reply db/projects/ost/project.yaml]: linkedin/company: ostdotcom

[the remark this answers is not in the document]
Comment (cryptomari0) [reply db/projects/ost/project.yaml]: location:hq: 'Hong Kong'

File: db/projects/ost/project.yaml [hunk 4]
```diff
@@ -0,0 +1,96 @@
+name: OST
```
[the remark this answers is not in the document]
Comment (cryptomari0) [reply db/projects/ost/project.yaml]: done

File: db/projects/red-pulse-phoenix/project.yaml [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+  github: RedPulse
+  bitcointalk: '2096780.0'
+  telegram: redpulse
+  twitter: red_pulse_china
+  reddit: RedPulseToken
+  linkedin/company: 'red-pulse'
+  youtube/channel: UC8KBaSywi1AhmKXTT3MqClA
+contacts:
+  email: 'info@redpulse.com'
+  location:hq: 'Rooms 05 -15, 13A/F, South Tower, World Finance Centre, Harbour City, 17 Canton Road, Tsim Sha Tsui, Kowloon, Hong Kong'
```
[the remark this answers is not in the document]
Comment (cryptomari0) [reply db/projects/red-pulse-phoenix/project.yaml]: location:hq: 'Kowloon, Hong Kong'
  location:node: 'Shanghai, China'

I do not know, in other hand, if there would be full addresses, we could have in the future also maps...

File: db/projects/request-network/project.yaml [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+    photo: https://request.network/assets/img/team/JDE.png
+    webids:
+      linkedin: juliendevoir
+      twitter: JulienDevoir
+whitepapers:
+  - title: 'Whitepaper: Request Network: The future of commerce: A decentralized network for payment requests'
+    url: https://request.network/assets/pdf/request_whitepaper.pdf
+tags:
+  - payment
+  - ivoice
```
[the remark this answers is not in the document]
Comment (cryptomari0) [reply db/projects/request-network/project.yaml]: - invoice

File: db/projects/request-network/project.yaml [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+    webids:
+      linkedin: 'laura-girod-99315b97'
+  - name: Julien Devoir
+    role: cmo
+    photo: https://request.network/assets/img/team/JDE.png
+    webids:
+      linkedin: juliendevoir
+      twitter: JulienDevoir
+whitepapers:
+  - title: 'Whitepaper: Request Network: The future of commerce: A decentralized network for payment requests'
```
[the remark this answers is not in the document]
Comment (cryptomari0) [reply db/projects/request-network/project.yaml]: - title: 'Request Network: The future of commerce: A decentralized network for payment requests'

File: db/projects/salt/project.yaml [hunk 8]
```diff
@@ -0,0 +1,40 @@
+name: SALT
```
[the remark this answers is not in the document]
Comment (cryptomari0) [reply db/projects/salt/project.yaml]: done

File: db/projects/singulardtv/project.yaml [hunk 9]
```diff
@@ -0,0 +1,45 @@
+name: SingularDTV
```
[the remark this answers is not in the document]
Comment (cryptomari0) [reply db/projects/singulardtv/project.yaml]: done

File: db/projects/request-network/project.yaml (hunk 7 above)
[the remark this answers is not in the document]
Comment (coinmandeer) [reply db/projects/request-network/project.yaml]: dvojita dvoutecka? :) To zni zajimave .. je to sice kravina, ale nebylo by lepsi tam mit "Request Network: Decentralized Network for Payment Request" ?

File: db/projects/polymath/project.yaml [hunk 10]
```diff
@@ -0,0 +1,44 @@
+name: Polymath
```
[the remark this answers is not in the document]
Comment (cryptomari0) [reply db/projects/polymath/project.yaml]: asset-poly-logo-square.png

File: db/projects/populous/project.yaml [hunk 11]
```diff
@@ -0,0 +1,70 @@
+name: Populous
```
[the remark this answers is not in the document]
Comment (cryptomari0) [reply db/projects/populous/project.yaml]: asset-poly-logo-full.png

[the remark this answers is not in the document]
Comment (cryptomari0) [reply db/projects/populous/project.yaml]: Actually asset-ppt-logo-full.png

File: db/projects/ravencoin/project.yaml [hunk 12]
```diff
@@ -0,0 +1,77 @@
+name: Ravencoin
```
[the remark this answers is not in the document]
Comment (cryptomari0) [reply db/projects/ravencoin/project.yaml]: asset-rvn-logo-full

File: db/projects/ravencoin/project.yaml [hunk 13]
```diff
@@ -0,0 +1,77 @@
...
+    start_date: '2018-01-03'
+    webids:
+      coinmarketcap: ravencoin
+      cryptocompare: rvn
+      cryptocompare/id: '794350'
+whitepapers:
+  - title: 'Ravencoin: A Peer to Peer Electronic System for the Creation and Transfer of Assets'
+    url: http://ravencoin.org/Ravencoin.pdf
+tags:
+  - westeros
```
[the remark this answers is not in the document]
Comment (cryptomari0) [reply db/projects/ravencoin/project.yaml]: tags:
  - transfer
  - opensource

**Review by coinmandeer-zz — DISMISSED** [review]

File: db/projects/red-pulse-phoenix/project.yaml [hunk 14]
```diff
@@ -0,0 +1,65 @@
+name: Red Pulse Phoenix
```
[the remark this answers is not in the document]
Comment (cryptomari0) [reply db/projects/red-pulse-phoenix/project.yaml]: asset-phx-logo-full.svg
asset-phx-logo-square.png

File: db/projects/request-network/project.yaml (hunk 7 above)
[the remark this answers is not in the document]
Comment (cryptomari0) [reply db/projects/request-network/project.yaml]: - title: 'Request Network: Decentralized Network for Payment Request'

**Review by coinmandeer-zz — APPROVED** [review]

**Pull request merged**